src/coreclr/src/jit/compiler.h

```diff
@@ -6512,7 +6512,7 @@ class Compiler
     void optCopyProp(BasicBlock* block, Statement* stmt, GenTree* tree, LclNumToGenTreePtrStack* curSsaName);
     void optBlockCopyPropPopStacks(BasicBlock* block, LclNumToGenTreePtrStack* curSsaName);
     void optBlockCopyProp(BasicBlock* block, LclNumToGenTreePtrStack* curSsaName);
-    bool optIsSsaLocal(GenTree* tree);
+    unsigned optIsSsaLocal(GenTree* tree);
```

Comment:
probably could have been renamed from an "Is" to "Get", but this is fine for the 5.0 port